src/managers/input/Tablets.cpp

```diff
@@ -159,10 +159,11 @@ void CInputManager::onTabletAxis(CTablet::SAxisEvent e) {
 void CInputManager::onTabletTip(CTablet::STipEvent e) {
     const auto PTAB  = e.tablet;
     const auto PTOOL = ensureTabletToolPresent(e.tool);
+    const auto POS = e.tip;
+    g_pPointerManager->warpAbsolute(POS, PTAB);
+    refocusTablet(PTAB, PTOOL, true);
 
     if (e.in) {
-        simulateMouseMovement();
-        refocusTablet(PTAB, PTOOL);
         PROTO::tablet->down(PTOOL);
     } else
```

Comment:
Remove the `{}` around the if to make CI happy again.

Reply:
the brace isnt why c-f fails, but yes, please nuke it and clang-format xD